src/components/cards/BasicListCard.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import styled from 'styled-components';
+import { Colors } from 'lattice-ui-kit';
+
+const { NEUTRAL } = Colors;
+
+const BasicListCard = styled.div`
```

Comment:
No change required. It would be neat to combine this component and the ones used in other places with similar design, though I'm not sure how easy it would be to combine.

Reply:
well this styled component is being used elsewhere, but would have to think more about it if we wanted to move the title (with appropriate icon) and optional buttons and onClick features.